paper/paper.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "CDP: Software for Municipal Data Collection, Analysis, and Publication"
+tags:
+    - Python
+    - JavaScript
+    - open government
+    - open data
+    - open infrastructure
+    - municipal governance
+    - data archival
+    - civic technology
+    - natural language processing
+authors:
+    - name: Jackson Maxfield Brown
+      orcid: 0000-0003-2564-0373
+      affiliation: 1
+    - name: To Huynh
+      orcid: 0000-0002-9664-3662
+      affiliation: 2
+    - name: Isaac Na
+      orcid: 0000-0002-0182-1615
+      affiliation: 3
+    - name: Brian Ledbetter
+      affiliation: 1
+    - name: Hawk Ticehurst
+      affiliation: 1
+    - name: Sarah Liu
+      affiliation: 4
+    - name: Emily Giles
+      affiliation: 4
+    - name: Katlyn Greene
+      affiliation: 4
+    - name: Sung Cho
+      affiliation: 4
+    - name: Shak Ragoler
+      affiliation: 4
+    - name: Nicholas Weber
+      orcid: 0000-0002-6008-3763
+      affiliation: 1
+
+affiliations:
+    - name: University of Washington iSchool, University of Washington, Seattle
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: Washington University, St. Louis
+      index: 3
+    - name: Independent Contributor
+      index: 4
+
+date: 29 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Cities, counties, and states throughout the USA are bound by law to archive recordings of public meetings. Most local governments comply with these laws by posting documents, audio, or video recording online. Council Data Project (CDP) is three open-source tools that improve the accessibility of local government data by systematically collecting, transforming, and re-publishing this data to the web. The software provided here can be used by civic technologists to promote government transparency, or using the processed data in support of research in public policy, natural language processing and machine learning, or information retrieval and discovery.
```

Comment:
> "Council Data Project (CDP) is a project that improves the accessibility..." since later on, you expand upon the fact the CDP is comprised of three primary tools in the architecture section.

I hear this but also two "projects" in the same sentence basically right after one another reads poorly imo. I am going to change it to "Council Data Project (CDP) is a set of open-source tools" leaving it a bit more open ended.

> The last sentence also feels a bit clunky since you're jumping between different tenses (i.e. "used by", "using").
> 
> Alternative suggestion:
> 
> This re-published data is packaged and presented within a searchable web application that vastly simplifies the process of finding specific information within the archived data. We envision this project being used by a variety of groups including civic technologists hoping to promote government transparency, researchers focused on public policy, natural language processing, machine learning, information retrieval and discovery, and many others.
> 
> ^ Still could use more rounds of editing/revision, but hopefully, that spurs some ideas.

Agree added with minor change.